guides/tutorials/codelabs/pubsub-to-appengine/index.md

```diff
@@ -140,6 +140,8 @@ Now configure Spinnaker to receive messages from your Google Cloud Pub/Sub subsc
 
 ### Deploy Spinnaker with Halyard
 
+Select a Spinnaker version: `hal config version edit --version <version>`. List the available versions with `hal version list`.
```

Comment:
Is this really needed? We don't add any other hal commands that aren't stricly required by this codelab (for example we don't configure you gce provider).